scene/get-elevation-at-a-point/README.md

```diff
@@ -1,22 +1,22 @@
-# Get elevation at a point
+# Get elevation at point
```

Comment:
reverting back to original to match sample name (and it fits well with the upcoming readme title standards)
```suggestion
# Get elevation at a point
```